scalding-core/src/main/scala/com/twitter/scalding/ReduceOperations.scala

```diff
@@ -111,7 +111,7 @@ trait ReduceOperations[+Self <: ReduceOperations[Self]] extends java.io.Serializ
     def log2(x: Double) = scala.math.log(x) / scala.math.log(2.0)
     val bits = 2 * scala.math.ceil(log2(104) - log2(errPercent)).toInt
     implicit val hmm = new HyperLogLogMonoid(bits)
-    mapPlusMap(f) { (t: T) => hmm(t) } (fn)
+    mapPlusMap(f) { (t: T) => hmm.create(t) } (fn)
```

Comment:
`HyperLogLogMonoid.apply` is deprecated.

Here's the definition: `def apply[T <% Array[Byte]](t: T) = create(t)`